src/components/tooltip/TooltipDirective.js

```diff
@@ -442,7 +442,7 @@ goog.require('ga_window_service');
                 var shopLayer = config.shop && !config.shopMulti;
                 var shopMultiLayer = config.shopMulti;
 
-                var limit = shopMultiLayer ? 10 : null;
+                var limit = shopMultiLayer ? 10 : 20;
```

Comment:
You can delete the `shopMultiLayer `  and `limit`  variable and put 10 directly in the gaIdentify call 2 lines further